ui/component/or-app/src/or-offline.ts

```diff
@@ -0,0 +1,107 @@
+import {Page} from "./types";
+import {css, html, TemplateResult} from "lit";
+import {customElement, state} from "lit/decorators.js";
+import {AppStateKeyed} from "./app";
+import {Store} from "@reduxjs/toolkit";
+import {i18next} from "@openremote/or-translate";
+import manager, {EventCallback, OREvent} from "@openremote/core";
+import {asyncReplace} from 'lit/directives/async-replace.js';
+import {when} from 'lit/directives/when.js';
+
+export function getPageOffline(store: Store<AppStateKeyed>): Page<AppStateKeyed> {
+    return new OrOffline(store);
+}
+
+// language=css
+const styling = css`
+    #offline-wrapper {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100%;
+        width: 100%;
+        flex-direction: column;
+        gap: 32px;
+    }
+
+    #offline-icon {
+        font-size: 64px;
+    }
+
+    #offline-text-container {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        gap: 16px;
+    }
+
+    #offline-title {
+        font-size: 24px;
+        font-weight: bold;
+    }
+`
+
+async function* countDown(count: number) {
+    while (count > 0) {
+        yield count--;
+        await new Promise((r) => setTimeout(r, 1000));
+    }
+}
+
+
+@customElement("or-offline")
```

Comment:
Maybe put page in the name of this, given webcomponent names must be globally unique the more unique it is the better chance of avoiding conflict with a custom component